src/frontend/efiling-frontend/src/components/page/upload/__snapshots__/Upload.stories.storyshot

```diff
@@ -55,14 +55,14 @@ exports[`Storyshots Upload Default 1`] = `
       >
         <button
           class="bcgov-button bcgov-normal-white btn"
-          data-testid="cancel-upload-btn"
+          data-test-id="cancel-upload-btn"
```

Comment:
Remember to always do a `yarn install` after a pull from master.
This looks like you are reverting the snapshots we recently fixed with the updated to shared-components.
`<Button />` now uses `data-testid`.

https://github.com/bcgov/react-shared-components/blob/a1e8a74bae682256c8926bd3cb95dae435370eb5/src/components/button/Button.js#L19